src/modules/endpoint/models/CategoryModel.js

```diff
@@ -69,6 +73,14 @@ class CategoryModel {
   get availableLanguages (): Map<string, string> {
     return this._availableLanguages
   }
+
+  getLastUpdate (locale: string): Moment {
+    const lastUpdate = this._lastUpdate
+    lastUpdate.locale(locale)
+
+    // only return day, month and year
+    return lastUpdate.format('LL')
+  }
```

Comment:
Um actually... :D I think this toString conversion should be made within the TimeStamp component. So we give the timeStamp a moment-object as prop